src/shared/approval-status/get-approval-status.test.js

```diff
@@ -23,13 +25,16 @@ describe('getApprovalStatusDisplayData', () => {
         })
     })
     it('returns the correct display data for approval status "ACCEPTED_HERE"', () => {
+        const yearDiff =
+            new Date().getFullYear() - new Date(BASELINE_DATE_STR).getFullYear()
+
         expect(
             getApprovalStatusDisplayData({
                 approvalStatus: APPROVAL_STATUSES.ACCEPTED_HERE,
                 approvedBy: 'Hendrik',
             })
         ).toEqual({
-            displayName: 'Approval by Hendrik accepted 2 years ago',
+            displayName: `Approval by Hendrik accepted ${yearDiff} years ago`,
```

Comment:
FYI: this test failure, was actually due to the fact that we've progressed one year in time